.circleci/scripts/setup_ci_environment.sh

```diff
@@ -27,7 +27,7 @@ docker version
 retry sudo pip -q install awscli==1.16.35
 
 if [ -n "${USE_CUDA_DOCKER_RUNTIME:-}" ]; then
-  DRIVER_FN="NVIDIA-Linux-x86_64-450.51.06.run"
```

Comment:
https://s3.amazonaws.com/ossci-linux/nvidia_driver/NVIDIA-Linux-x86_64-455.32.00.run doesn't actually exist yet

Reply:
Uploaded `NVIDIA-Linux-x86_64-455.38.run` to the above-mentioned S3 bucket